portfolio/src/main/webapp/skills.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <meta charset="UTF-8">
+        <title>Ross Johnson's Skills</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <center>
+            <h1>
+                Ross Johnson's Skills and Previous Experience
+            </h1>
+            <img id="interestimg" src="./images/flutter.png"></img>
```

Comment:
generally, each id should be unique.

Reply:
To tag on, seeing this is used for CSS. This could be done using a class rather than an id.